backend/views.py

```diff
@@ -2,8 +2,12 @@
 
 from app import oidc
 
+from flask import current_app
```

Comment:
Instead of always using the `current_app` to access the logger, I would suggest to put the logger in the global variable in app.py and import that global logger variables instead. Please refer to line #3 in this file as how we are importing and using oidc.